gradio/blocks.py

```diff
@@ -910,7 +914,7 @@ def render(self):
             dependency_offset = len(Context.root_block.dependencies)
             for i, dependency in enumerate(self.dependencies):
                 api_name = dependency["api_name"]
-                if api_name is not None:
+                if api_name is not None and not False:
```

Comment:
Is the intention to not run this code block if `api_name` is False? If so, I don't think this is working correctly. `not False` always evaluates to `True`:

<img width="411" alt="image" src="https://github.com/gradio-app/gradio/assets/41651716/5fd121b9-c3e0-44b0-a628-acfed308b2e4">

Reply:
Lol that should be: `if api_name is not None and api_name is not False`